### PR TITLE
MTM-44126 Added related topics to Getting started

### PR DESCRIPTION
**Pull request by MWindel**
Added related topics to the Getting started section in the User guide.

### CODE REVIEW

**Review by rssag — APPROVED** [review]

File: content/users-guide/getting-started-bundle/platform-applications.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ aliases:
   - /users-guide/getting-started/#cumulocity-applications
 ---
 
+{{< c8y-admon-related >}}
+- [Administration > Managing applications](/users-guide/administration/#managing-applications) for more information on managing applications and microservices in {{< product-c8y-iot >}}.
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/platform-applications.md:11]:
```suggestion
- [Administration > Managing applications](/users-guide/administration/#managing-applications) for details on the available applications and information on managing applications in {{< product-c8y-iot >}}.
```

File: content/users-guide/getting-started-bundle/gui-features.md [hunk 2]
```diff
@@ -6,7 +6,11 @@ aliases:
   - /users-guide/overview/#gui-features
 ---
 
-
+{{< c8y-admon-related >}}
+- [Cockpit](/users-guide/cockpit/#overview) for more details on the {{< product-c8y-iot >}} UI.
+- [Administration > Managing applications](/users-guide/administration/#managing-applications) for more information on managing applications and microservices in {{< product-c8y-iot >}}.
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/gui-features.md:11]: I´m not sure if this is really relevant here. Btw, Managing applications is only about applications, not microservices (there is an extra chapter for this).
I would definitely add a link to the Web SDK documentation.

File: content/users-guide/getting-started-bundle/gui-features.md [hunk 3]
```diff
@@ -6,7 +6,11 @@ aliases:
...
+- [Device Management](/users-guide/device-management/#overview) for more information on working with devices in the {{< product-c8y-iot >}} UI.
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/gui-features.md:12]: I would remove this, as this chapter is about UI functionalities in general?

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@eniosultan Hi Enio, we would like to add some "related topics" at the top of each chapter to guide the users to other places of interest in the documentation. We have already added what we found useful but perhaps you have some more/better ideas? Could you please review and comment if you think we should add links? Thanks!

**Review by eniosultan — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**